SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/SpatialWorkerFlags/DummyObject.cpp

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "DummyObject.h"
+
+void  UDummyObject::SetFlagUpdated(FString flagName, FString flagValue)
```

Comment:
can we make these parameters const&, we may need to update the delegates for this.